src/main/scala/com/microsoft/hyperspace/index/rules/JoinIndexRule.scala

```diff
@@ -234,9 +236,6 @@ object JoinIndexRule
    *    WHERE newT1.aliasCol = T2.b
    * Here, aliasCol is not directly from the base relation T1
    *
-   * TODO: add alias resolver for supporting aliases in join condition. Until then,
```

Comment:
which change is handling the aliases?

Reply:
add a test if this is fixed?

Reply:
I thought this was the case https://github.com/microsoft/hyperspace/issues/25 and we decided not to fix it until 3.0?

Reply:
Got it.

In this case, can we add something like `The following scenario will be supported in Spark 3.0 (SPARK-30298)` for the above block (starting with `This will not be true`)?

Reply:
Let me revert alias related changes and take it in a separate PR